test-project/build.gradle

```diff
@@ -7,6 +7,14 @@ def setupGraalDependencies(String platform) {
         throw new GradleException("Platform $platform not supported")
     }
 
+    def skipGraalvmHome = Boolean.getBoolean("skipGraalvmHome")
```

Comment:
I would make this a gradle property (so it will be provided with `-P`). That way it's more clear that this is a gradle only thing and not part of substrate itself?

Reply:
ok, make sense

Reply:
What is the goal of this property?